frontend/src/helpers/user-service/api-wrapper.ts

```diff
@@ -0,0 +1,87 @@
+import axios from "axios";
+import { UserInfo } from "../../components/common/ReadOnlyUserCard";
```

Comment:
Kindly extract this out from the component to your user-service.d.ts

Reply:
Resolved